guides/v2.2/config-guide/config/disable-module-output.md

```diff
@@ -78,8 +89,8 @@ Archive the original `<Magento_install_dir>/app/etc/config.php` file. Then add t
 
 Here:
 
- - The `array` beneath `modules_disable_output` contains a list of modules. 
- - `Magento_Review` specifies which module to disable output for.
- - `1` is the flag that disables output for the `Magento_Review` module.
+- The `array` beneath `modules_disable_output` contains a list of modules.
+- `Magento_Review` specifies which module to disable output for.
+- `1` is the flag that disables output for the `Magento_Review` module.
```

Comment:
I would rewrite lines 93-94 to something like 

In this example, output for the `Magento_Review` module has been disabled. To re-enable output, set the value to `0`.

Reply:
Got it.